aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -185,6 +189,23 @@ module aptos_framework::code {
             request_publish(addr, module_names, code, policy.policy)
     }
 
+    public fun is_package_upgradeable(metadata_serialized: vector<u8>): bool {
```

Comment:
Could we get some doc comments here? As it is now, based on the signature, I have no idea what bytes I'm meant to be passing in.

Reply:
Discussed with Kevin, will remove this as we are no longer check this.